R/datatostone.h.R

```diff
@@ -118,11 +118,11 @@ dataTOSToneResults <- R6::R6Class(
                     list(`name`="t[0]", `title`="t", `type`="number"),
                     list(`name`="df[0]", `title`="df", `type`="integer"),
                     list(`name`="p[0]", `title`="p", `type`="number", `format`="zto,pvalue"),
-                    list(`name`="b[1]", `title`="", `type`="text", `content`="TOST Upper"),
+                    list(`name`="b[1]", `title`="", `type`="text", `content`="TOST Lower"),
                     list(`name`="t[1]", `title`="t", `type`="number"),
                     list(`name`="df[1]", `title`="df", `type`="integer"),
                     list(`name`="p[1]", `title`="p", `type`="number", `format`="zto,pvalue"),
-                    list(`name`="b[2]", `title`="", `type`="text", `content`="TOST Lower"),
+                    list(`name`="b[2]", `title`="", `type`="text", `content`="TOST Upper"),
```

Comment:
just FYI, these `.h.R` files are automatically generated from the `.r.yaml` files (mentions this at the top of the file) ... so any changes to these files will be subsequently overwritten. so the `.r.yaml` files need to be updated too (i'll submit a PR shortly).